datadog_checks_base/tests/stubs/test_aggregator_similar.py

```diff
@@ -214,3 +218,20 @@ def test__build_similar_elements__histogram_buckets(self, aggregator):
         assert similar_histogram_bucket[3][1].name == 'histogram.bucket4'  # value/monotonic/tag match
         assert similar_histogram_bucket[4][1].name == 'histogram.bucket5'  # value/monotonic match
         assert similar_histogram_bucket[5][1].name == 'histogram.bucket0'  # no match
+
+    # Test diff methods used in showing the difference to the closest match
+    def test_dict_diff(self):
+        assert similar.dict_diff({'a': 1, 'b': 2}, {'a': 1, 'b': 3}) == [('b', 2, 3)]
+        assert similar.dict_diff({'a': 1}, {'b': 3}) == [('a', 1, None)]
+        assert similar.dict_diff({'a': 1}, {'a': 1}) == []
+        assert similar.dict_diff({'a': 1}, {}) == [('a', 1, None)]
+        assert similar.dict_diff({}, {'a': 1}) == []
+        assert similar.dict_diff({}, {}) == []
```

Comment:
Please either split these up or use `pytest.parametrize` with test IDs that describe what's being tested.

E.g.: `{'a': 1, 'b': 2}, {'a': 1, 'b': 3}` -> `key present but with different value`, `{}, {'a': 1}` -> `extra key in closest match`.